src/components/Question/Question.js

```diff
@@ -109,26 +119,70 @@ class Question extends Component {
     });
   }
 
+  handleKeyDown(event) {
+    /* don't trigger navigation if the current url isn't a question */
```

Comment:
the current url should always be a question if you are adding the event listener onMount and removing onUnmount

Reply:
I was observing some weird behavior before I added that line; keyboard navigation was still kicking in even though I wasn't on a question page. I'm not sure what the cause of that was. I'll see if I can reproduce the bug.

Reply:
I would guess it was from not removing the event listener onUnmount

Reply:
Yeah, you're right. It was a result of me registering an anonymous function to the event listener and not being able to remove it. Whoops!